gpu-operator/google-gke.rst

```diff
@@ -67,11 +67,115 @@ Prerequisites
   in the Google Cloud documentation.
 
 
-*********
-Procedure
-*********
+*********************************
+Using the Google Driver Installer
```

Comment:
In terms of structure, I am wondering if this page would be easier to follow if the section headers aligned with the bullets from the introduction:
```
Using Container-Optimized OS
...

Using Ubuntu
...
```
For each OS, we provide the recommended procedure. For COS, that would be Google's driver installer + GPU Operator for the rest. For Ubuntu, that would be GPU Operator for everything. 

One downside -- if we wanted to document multiple options for each OS, i.e. documenting two procedure for Ubuntu (Google vs NVIDIA managed drivers), that would lead to some duplication across the various procedures.

Reply:
We discussed this offline and decided the current structure will suffice.